source/DemoScene.cpp

```diff
@@ -80,5 +85,16 @@ bool DemoScene::update(Game &)
   if (Keyboard::getKeys()[OIS::KC_S]) {
     cameraNode->translate(0, 0, 10);
   }
+
+  if (Keyboard::getKeys()[OIS::KC_U]) {
+    illidan.stopAnimation();
+  }
+  if (Keyboard::getKeys()[OIS::KC_O]) {
+    illidan.setAnimation("Move", false);
+  }
+  if (Keyboard::getKeys()[OIS::KC_P]) {
+    illidan.setAnimation("Stand", false);
+  }
+  illidan.updateAnimations(fe.timeSinceLastFrame);
```

Comment:
The order shouldn't change anything.

If sometimes you have a little glitch and the illidan is doing the first movement of an other animation, it's from the model, not the code. I did the animation a bit quickly, and some animations are overlapping on their first / last frames. This will be fix when i'll redo illidan's model exportation, when the game will be more mature.